File: Exercicio-05.c
#include <stdio.h>

int main(void) {

  int n1, n2, n3;

  printf("Digite o primeiro valor: ");
  scanf("%d", &n1);

  printf("Digite o segundo valor: ");
  scanf("%d", &n2);
  
  printf("Digite o terceiro valor: ");
  scanf("%d", &n3);
  
  if (n1 > n2 && n1 > n3){
      printf("O primeiro numero [%d] é maior!", n1);
  } else if (n2 > n1 && n2 > n3){
      printf("O segundo numero [%d] é maior!", n2);
  } else{
      printf("O terceiro numero [%d] é maior!", n3);
  }
  return 0;
}

File: Exercicio-05.java
import java.util.Scanner;

public class Main
{
  public static void main (String[]args)
  {
    Scanner sc = new Scanner (System.in);
    int n1, n2, n3;

    System.out.println("Digite o primeiro valor: ");
    n1 = sc.nextInt();

    System.out.println("Digite o segundo valor: ");
    n2 = sc.nextInt();
    
    System.out.println("Digite o terceiro valor: ");
    n3 = sc.nextInt();
    
    if (n1 > n2 && n1 > n3){
      System.out.printf("O primeiro numero [" + n1 + "] é maior!");
  } else if (n2 > n1 && n2 > n3){
      System.out.printf("O segundo numero [" + n2 + "] é maior!");
  } else{
      System.out.printf("O terceiro numero [" + n3 + "] é maior!");
  }
  }
}
